ballerina/tests/test.bal

```diff
@@ -140,6 +165,28 @@ function testCreate() {
     }
 }
 
+@test:Config {
+    enable: true
+}
+function testQueryPasswordFlow() returns error? {
+    log:printInfo("baseClientPasswordFlow -> query()");
+    string sampleQuery = "SELECT name FROM Account";
+    stream<Account, error?> queryResult = check baseClientPasswordFlow->query(sampleQuery);
+    int count = check countStream(queryResult);
+    test:assertTrue(count > 0, msg = "Found 0 search records!");
+}
+
+@test:Config {
+    enable: true
+}
+function testQueryCredentialsFlow() returns error? {
+    log:printInfo("baseClientCredentialsFlow -> query()");
+    string sampleQuery = "SELECT name FROM Account";
```

Comment:
we can pass the query inline here

Reply:
updated in [8385ec2](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/302/commits/8385ec2ccf542c86b49f917b37d071f806bc0341)